docs/modules/ROOT/pages/concepts/testing_applications/sanity_tests.adoc

```diff
@@ -1,29 +1,29 @@
-= Sanity Tests
+= Surface-level tests
 CP Development Team <cp-devel@redhat.com>
 :toc: left
 :icons: font
 :numbered:
 :source-highlighter: highlightjs
 
 == Scope
-This document covers the sanity tests contained in {ProductName} that are executed as part of the {ProductName} component build pipeline. The sanity tests for {ProductName} automatically test all application images to ensure they are up to date, correctly formatted, and protect them from security vulnerabilities.
+This document covers the surface-level tests (formerly known as "sanity tests") that {ProductName} runs as part of its component build pipeline. These surface-level tests automatically check all application images to ensure that they're up-to-date, correctly formatted, and protected from security vulnerabilities.
```

Comment:
Hm, looking at the `protected from security vulnerabilities` again, this might be misleading, since the security vulnerabilities that surface level tests protect from are, well, very much surface level :smile:

Reply:
i made that change, @dirgim and created this [new PR](https://github.com/redhat-appstudio/docs.stonesoup.io/pull/29). hope this was OK.

Reply:
Sure, it's ok with me.